### ISSUE TITLE
error in Change focus + direction in max layout

### ISSUE THREAD

**Issue by przymusp** [Elv13/collision#27]
## Description

I get error with collison when changing focus in max layout, any guess what is wrong?

## Stack traceback
 
``` 
2019-08-21 11:46:29 W: awesome: luaA_dofunction:78: error while running function!
stack traceback:
	/usr/share/awesome/lib/gears/shape.lua:765: in local 's'
	/home/eror/.config/awesome/collision/max.lua:172: in local 'icon_f'
	/home/eror/.config/awesome/collision/max.lua:113: in upvalue 'draw_shape'
	/home/eror/.config/awesome/collision/max.lua:216: in function 'collision.max.display_clients'
	/home/eror/.config/awesome/collision/init.lua:98: in function 'collision.focus'
	/home/eror/.config/awesome/collision/init.lua:158: in upvalue 'press'
	/usr/share/awesome/lib/awful/key.lua:125: in function </usr/share/awesome/lib/awful/key.lua:125>
error: /usr/share/awesome/lib/gears/shape.lua:765: attempt to call a nil value (upvalue 'shape')
```

## System info
* collison master 976aa2051f1560bfa2ef0b71eb481fb5d968c916
* awesome v4.3 (Too long) 
* Compiled against Lua 5.3.3 (running with Lua 5.3)
* config based on awesome-copycats

**Comment by przymusp** [comment]
Additionally i get a black bar at bottom of screen (supposedly for clients preview ? ), and it hangs until awesome restart.

**Comment by przymusp** [comment]
The same problem occurs with default config for awesome (from debian) with collision included (with removed default mod + arrow bindings).

**Comment by przymusp** [comment]
fixed in #28
